src/main/java/org/jenkinsci/plugins/ewm/steps/ExwsExecution.java

```diff
@@ -108,14 +107,14 @@ public boolean start() throws Exception {
 
     /**
      * TODO JAVADOC
+     *
      * @param workspaceId
      * @throws IOException
      */
     private void registerFingerprint(String workspaceId) throws IOException {
-        FingerprintMap map = Jenkins.getActiveInstance().getFingerprintMap();
-        Fingerprint f = map.get(workspaceId);
+        Fingerprint f = Jenkins.getActiveInstance()._getFingerprint(workspaceId);
         if (f == null) {
-            return;
+            throw new AbortException("Couldn't find any Fingerprint for: " + workspaceId);
```

Comment:
Is it useful for the user to know the full workspace path if this occurs? If yes, maybe it is a good idea to catch the exception in the `start()` method and give more info to the user. If no, then ignore this comment.